src/NosCore.MasterServer/Controllers/WarehouseController.cs

```diff
@@ -51,7 +54,7 @@ public List<WarehouseLink> GetWarehouseItems(Guid? id, long? ownerId, WarehouseT
             var list = new List<WarehouseLink>();
             if (id == null)
             {
-                var warehouse = _warehouseDao.FirstOrDefault(s
+                var warehouse = _warehouseDao.FirstOrDefaultAsync(s
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'warehouse'.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5326532)